packages/auth/src/accessToken/service.ts

```diff
@@ -72,16 +86,39 @@ async function introspect(
   }
 }
 
-async function revoke(
-  deps: ServiceDependencies,
-  id: string
-): Promise<Error | undefined> {
+async function revoke(deps: ServiceDependencies, id: string): Promise<void> {
   const token = await AccessToken.query(deps.knex).findById(id)
   if (token) {
-    if (!isTokenExpired(token)) {
-      await token.$query(deps.knex).patch({ expiresIn: 1 })
+    await token.$query(deps.knex).delete()
+  }
+}
+
+async function rotate(
+  deps: ServiceDependencies,
+  managementId: string
+): Promise<Rotation> {
+  let access: Access | undefined
+  let error: Error | undefined
+
+  const token = await AccessToken.query(deps.knex).findOne({ managementId })
+  if (token) {
+    await token.$query(deps.knex).patch({
+      value: uuid()
+    })
```

Comment:
Note that @njlie pointed out that the `managementId` SHOULD be updated, too.

Reply:
After reading the spec again, I think we should just delete the old token and create a new one.